modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v2/structure.rb

```diff
@@ -27,7 +29,7 @@ def form_fill
             form_fields.virtual_tele_hearing => form_data.virtual_tele_hearing,
             form_fields.requesting_extension => form_data.requesting_extension,
             form_fields.appealing_vha_denial => form_data.appealing_vha_denial,
-            form_fields.additional_issues => form_data.additional_pages,
+            form_fields.additional_issues => form_data.extra_contestable_issues,
```

Comment:
THis is the bug where we were checking the extra issues checkbox for any overflow pages, even if the overflow page are for long emails or long rep names.

Changed it to check the box only if there are more than 5 issues and an issues overflow page is added